### ISSUE TITLE
Support for adding or modifying nodes?

### ISSUE THREAD

**Issue by postmodern** [YorickPeterse/oga#110]
Can Oga add new nodes, copy existing ones or modify attributes/inner-text?

**Comment by YorickPeterse** [comment]
Yes, this is covered in the following documents:

* http://code.yorickpeterse.com/oga/latest/#examples
* http://code.yorickpeterse.com/oga/latest/file.manually_creating_documents.html

The documentation of the various Node classes (e.g. http://code.yorickpeterse.com/oga/latest/Oga/XML/Element.html) and the NodeSet class (http://code.yorickpeterse.com/oga/latest/Oga/XML/NodeSet.html) covers the various methods you'd use for this.

In short, adding elements is as easy as:

    some_element.children << Oga::XML::Element.new(:name => 'div')

Setting/getting attributes is done using:

    some_element.get('class')
    some_element.set('class', 'foo')

Removing elements:

    some_element.children.last.remove
